shared/arkitekt-decompose/src/commonMain/kotlin/app/futured/arkitekt/decompose/ArkitektComponentContext.kt

```diff
@@ -0,0 +1,10 @@
+package app.futured.arkitekt.decompose
```

Comment:
`arkitect-decompose` and `arkitekt-cr-usecases` packages has to be added to init_template.kts as packages to be renamed in `fun renamePackagesInShared` ☝️

Reply:
Why we should want to rename the `app.futured.arkitekt.*` package to the application one?

Reply:
@RudolfHladik https://github.com/futuredapp/kmp-futured-template/pull/102